src/FocusApp/FocusApp.csproj

```diff
@@ -64,6 +67,15 @@
 		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
 		<PackageReference Include="Sharpnado.Tabs.Maui" Version="3.0.1" />
+		<PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.1" />
```

Comment:
Since EF Core won't be implemented into the FocusApp in this issue, I think it would be best to remove it from the project.